src/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+
+import Page from './page';
+
+function App() {
+  const [toDoState, setToDoState] = useState({ index: 0, todo: [] });
+  const [inputState, setInputState] = useState({ inputText: '' });
+  const { inputText } = inputState;
+  const { todo } = toDoState;
+  function handleAddClick() {
+    const { index } = toDoState;
+    setInputState({ ...inputState.inputText, inputText: '' });
+    setToDoState({
+      ...toDoState,
+      index: index + 1,
+      todo: [...toDoState.todo, { index: index + 1, title: inputText }],
+    });
+  }
+  function handleChange(e) {
+    setInputState({ ...inputState, inputText: e.target.value });
+  }
+  function clickComplete(index) {
+    setToDoState({ ...toDoState, todo: [...todo.filter((v) => v.index !== index)] });
+  }
```

Comment:
```suggestion
import React, { useState } from 'react';
import ReactDOM from 'react-dom';

import Page from './page';

function App() {
  const [toDoState, setToDoState] = useState({ index: 0, todo: [] });
  const [inputState, setInputState] = useState({ inputText: '' });
  
  const { inputText } = inputState;  
  
  const { todo, index } = toDoState;
  
  function handleAddClick() {
    setInputState({ ...inputState.inputText, inputText: '' });
    
    setToDoState({
      ...toDoState,
      index: index + 1,
      todo: [
        ...toDoState.todo,
        { index: index + 1, title: inputText }
      ],
    });
  }
  
  function handleChange(e) {
    setInputState({ 
      ...inputState,
      inputText: e.target.value
     });
  }
  
  function clickComplete(index) {
    setToDoState({ 
      ...toDoState,
      todo: [...todo.filter((v) => v.index !== index)],
    });
  }
```

어떻게 의도를 드러내느냐에 따라 다르겠지만 이정도로 정리 해줘도 코드를 읽을 때 훨씬 좋을 것 같아요.